pages/index.tsx

```diff
@@ -1,14 +1,20 @@
 import Head from 'next/head';
+import { useRouter } from 'next/router';
 
 export default function Home() {
+  const router = useRouter();
   return (
     <div>
       <Head>
         <title>Create Next App</title>
         <meta name="description" content="Generated by create next app" />
         <link rel="icon" href="/favicon.ico" />
       </Head>
-      <main></main>
+      <main>
+        <button type="button" onClick={() => router.push('/SubmitTestimony')}>
```

Comment:
ah we renamed this url route in the previous PR since its best practice for urls to be `snake-case` and **not** `PascalCase`